tests/requirements-test.txt

```diff
@@ -1,3 +1,4 @@
 pytest==4.6.9
 pytest-selenium==1.17.0
 pystemd==0.8.0
+docker==4.4.0
```

Comment:
please sort the package names, it is easier to find a package when sorted

Reply:
Done